setup.py

```diff
@@ -105,12 +105,13 @@
         'lmfit>=1.0,<2',
         'requests>=2.21.0,<3',
         'colorama>=0.4.1,<1',
-        'pandas>=1.0,<2',
-        'scipy==1.4.1',
+        'pandas>=1.0',

```

Comment:
```suggestion
        'pandas>=1.0,<2',
```
Suggest to keep this as is, to protect us against possible API changes when pandas 2.0 eventually comes out.

Reply:
Fair point, I didn't consider that. Tho I doubt there will that much change during version updates as pandas is becoming a big backbone so backward compatibility would be a priority unless mentioned in update. Just an opinion tho

Reply:
I actually agree and removed the version pins from basically everything. They tend to cause more trouble than benefits when used with other packages.